packages/twenty-server/src/engine/core-modules/auth/utils/get-microsoft-apis-oauth-scopes.ts

```diff
@@ -6,6 +6,7 @@ export const getMicrosoftApisOauthScopes = () => {
     'offline_access',
     'Mail.Read',
     'Calendars.Read',
+    'User.Read',
```

Comment:
profile/email seems redundant with User.Read, is the the old/new way of doing seems with MS API? Seems like they changed the structure?

Reply:
TBO I inherietd from  
    'openid',
    'email',
    'profile',
    'offline_access',
    'Mail.Read',
    'Calendars.Read',
I only added the 'User.Read'
Not sure so far what is necessary on the API permissions for the Auth and for Calendar. Do you want me to investigate in order to  reduce the scopes ?

Reply:
As discussed with @charlesBochet, let's let that at the moment, and do the work to find what can be withdrawn later on if required by our users